app/views/admin/merchants/edit.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= form_for [@merchant], url: update_merchant_path(@merchant) do |f| %>
```

Comment:
Oooo, nice. Still wrapping my head around these,  but I like the way this looks. Very clean.